spec/models/linked_application_type_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe LinkedApplicationType do
+  describe ".link_types" do
```

Comment:
```suggestion
  describe ".all" do
```

Reply:
Possible tests to lock down the duck type and values

```ruby
it "returns a collection of Link types" do
   # expect(described_class.all).to all(be_instance_of(LinkTypeStruct))
   expect(described_class.all).to all(respond_to?(:code))
   expect(described_class.all).to all(respond_to?(:description))
end

it "returns expected collection attribute values" do
   expect(described_class.all).to match_array....
end
```

Reply:
👍 Had to tweak this suggestion slightly to get tests passing but have added tests for respond_to?(:code) and respond_to?(:description)